_dev/rescripts.js

```diff
@@ -5,8 +5,8 @@
 const { resolve } = require('path');
 
 const additionalJSImports = {
-  'fxa-react': __dirname,
-  'fxa-shared': resolve(__dirname, '../fxa-shared'),
+  'fxa-react': resolve(__dirname, '../packages/fxa-react'),
```

Comment:
im _pretty sure_ these could be like `require.resolve('fxa-react')`

Reply:
Looks like this resolves to `path/fxa-shared/dist/index.js`